firestore.rules

```diff
@@ -41,7 +41,7 @@ service cloud.firestore {
     match /repositories/{repository} {
       allow create: if false;
       allow get: if resource.data.private == false;
-      allow list: if false;
+      allow list: if true;
```

Comment:
I know you have not finished this part yet, but I think it should be  the same as line `16 for the user list

Reply:
okay will fix it.

Reply:
For me Line `12` worked.